nixos/modules/services/databases/redis.nix

```diff
@@ -166,7 +166,9 @@ in {
             save = mkOption {
               type = with types; listOf (listOf int);
               default = [ [900 1] [300 10] [60 10000] ];
-              description = "The schedule in which data is persisted to disk, represented as a list of lists where the first element represent the amount of seconds and the second the number of changes.";
+              description = "The schedule in which data is persisted to disk, represented as a list of lists where the first element represent the amount of seconds and the second the number of changes.
+
+              If set to the empty list (`[]`) then RDB persistence will be disabled (useful if you are using AOF or don't want any persistence).";
```

Comment:
the docs are not markdown unless wrapped in `mdDoc`. to be rendered properly this either needs wrapped or
```suggestion
              If set to the empty list (<literal>[]</literal>) then RDB persistence will be disabled (useful if you are using AOF or don't want any persistence).";
```

Reply:
Ok, I'll switch the the docbook unless you or another maintainer strongly prefers wrapping it for markdown.

Reply:
changing to docbook is fine for. though if you really want to you could apply https://github.com/NixOS/nixpkgs/pull/179483 to the module and convert all the docs :) (totally optional!)

Reply:
Let's keep that a separate change.

Reply:
I ended up using markdown after all because this is now two paragraphs and doing that in docbook is a hack that hurts me.

Reply:
Confirmed that this renders correctly:

![image](https://user-images.githubusercontent.com/494012/177319976-379967ce-1c7d-4091-a560-919c6bfb8185.png)